### PR TITLE
Associate tweets and articles

### PR DESCRIPTION
**Pull request by bmrsny**
Removed out logic from Articles controller so the controller no longer makes a call using Faraday. Instead moved this to a cron job that gets called. Added Twitter Relationship to Articles. Article has many Tweets and Tweets belong to an article. 
Added shoulda matchers gem. So please bundle
Also added column to the Tweet and Articles. So you will need to run rake db:migrate.

We now have tweets associated with our articles!!!! :)
:tada:

### CODE REVIEW

File: config/schedule.rb [hunk 1]
```diff
@@ -1,3 +1,6 @@
+set :environment, "development"
+set :output, {error: "log/cron_error_log.log", standard: "log/cron_log.log"}
```
Comment (bmrsny) [inline config/schedule.rb]: Removed Curly braces. Hope That was correct

**Comment by oorion** [comment]
Looks great to me.  Do all the tests pass?

**Comment by bmrsny** [comment]
yes

**Comment by mikedao** [comment]
:sparkles:

**Pull request merged**